tracer/src/Datadog.Trace/OTelMetrics/MetricReaderHandler.cs

```diff
@@ -155,41 +175,38 @@ private static bool IsValidInstrumentName(string name)
         return true;
     }
 
-    internal static AggregationTemporality? GetTemporality(InstrumentType instrumentType)
+    private static AggregationTemporality? TemporalityFor(InstrumentType kind, OtlpTemporality preference)
```

Comment:
nit: I think the previous name, or `GetAggregationTemporality`, sounds more conventional than `TemporalityFor`

```suggestion
    private static AggregationTemporality? GetTemporality(InstrumentType kind, OtlpTemporality preference)
```